chia/cmds/wallet.py

```diff
@@ -99,18 +99,19 @@ def show_cmd(wallet_rpc_port: int, fingerprint: int) -> None:
     asyncio.run(execute_with_wallet(wallet_rpc_port, fingerprint, {}, print_balances))
 
 
-@wallet_cmd.command("get_address", short_help="Get a wallet receive address")
+@wallet_cmd.command("get_address", short_help="Get the current public wallet address")
 @click.option(
     "-wp",
     "--wallet-rpc-port",
     help="Set the port where the Wallet is hosting the RPC interface. See the rpc_port under wallet in config.yaml",
     type=int,
     default=None,
 )
+@click.option( "-n", "--new_address", help="Generate new public wallet address.", type=bool, default=False, is_flag=True, show_default=True)
 @click.option("-i", "--id", help="Id of the wallet to use", type=int, default=1, show_default=True, required=True)
 @click.option("-f", "--fingerprint", help="Set the fingerprint to specify which wallet to use", type=int)
-def get_address_cmd(wallet_rpc_port: int, id, fingerprint: int) -> None:
-    extra_params = {"id": id}
+def get_address_cmd(wallet_rpc_port: int, id, new_address, fingerprint: int) -> None:
```

Comment:
```suggestion
def get_address_cmd(wallet_rpc_port: int, id, get_new_address: bool, fingerprint: int) -> None:
```

Reply:
i changed that righ away with the click option while syncing to the last branch. It also has to get changed in the click option